src/providers/twitch/TwitchIrcServer.cpp

```diff
@@ -37,6 +39,15 @@ TwitchIrcServer::TwitchIrcServer()
 
     this->pubsub = new PubSub(TWITCH_PUBSUB_URL);
 
+    if (getSettings()->enableSevenTVEventApi)
+    {
+        this->eventApi = new SeventvEventApi(SEVENTV_EVENTAPI_URL);
```

Comment:
Where is the ```cpp
delete```
to free the memory, why not just use a smartptr here?

Reply:
This is never deleted

Reply:
It is never deallocated because once running we don't stop it, the setting to disable it requires a restart.